homeassistant/components/jellyfin/config_flow.py

```diff
@@ -135,6 +135,45 @@ async def async_step_reauth_confirm(
             step_id="reauth_confirm", data_schema=REAUTH_DATA_SCHEMA, errors=errors
         )
 
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Allow users to update their Jellyfin configuration."""
+        errors: dict[str, str] = {}
+        entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])
+
+        if not entry:
+            return self.async_abort(reason="entry_not_found")
```

Comment:
You can use `self._get_reconfigure_entry()` for this

Reply:
I'm assuming you are referring to line 143 and 145-146. 

Looking at the code, I see that `_get_reconfigure_entry` calls `self.hass.config_entries.async_get_known_entry(self._reconfigure_entry_id)`, which is different than `async_get_entry` only because it will raise `raise UnknownEntry(entry_id)` is entry is None. So that will prevent the 145/146 path from ever being taken if entry is fasley, because the exception will be thrown instead.

Does some upstream mechanism handle the `UnknownEntry` exception in the same way as an abort return value?

Reply:
But the thing I'm missing is, why would the entry be None

Reply:
@Erotemic the idea is that it's not possible to start a reconfigure flow without a config entry, and if the config entry is removed while the reconfigure flow is in progress, the reconfigure flow will automatically be closed. Hence, the the helper `_get_reconfigure_entry` will always return a `ConfigEntry`, avoiding useless error handling in the reconfigure flow.